# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424025416 28800
#      Sun Feb 15 10:36:56 2015 -0800
# Node ID 1f66ff2a60a198d5ce093044ea84eefe69f70a68
# Parent  60ce9f86ecfe040f2f8c4ecd80ec9a38eca24866
Correct typos and English phrasings in GUI preferences dialog.

* settings-dialog.ui: Correct typos and English phrasings in GUI preferences
dialog.

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -347,17 +347,17 @@
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Octave Startup</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
-                 <string>These preferences are applied after the startup files like .octaverc.</string>
+                 <string>These preferences are applied after any .octaverc startup files.</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="5">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
@@ -402,17 +402,17 @@
                   <property name="text">
                    <string>Browse</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="cb_restore_octave_dir">
                   <property name="text">
-                   <string>Restore last Octave directory of previous session</string>
+                   <string>Restore working directory of previous session</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="le_octave_dir"/>
                 </item>
                </layout>
               </item>
@@ -694,17 +694,17 @@
                 </spacer>
                </item>
                <item row="6" column="0">
                 <widget class="QCheckBox" name="editor_long_line_marker">
                  <property name="toolTip">
                   <string>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</string>
                  </property>
                  <property name="text">
-                  <string>Draw a long  line marker</string>
+                  <string>Draw a long line marker</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QCheckBox" name="cb_code_folding">
@@ -724,26 +724,26 @@
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QCheckBox" name="cb_show_eol">
                  <property name="text">
-                  <string>Show eol characters</string>
+                  <string>Show EOL characters</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <widget class="QLabel" name="label_22">
                    <property name="text">
-                    <string>Default eol mode</string>
+                    <string>Default EOL mode</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="combo_eol_mode">
                    <property name="sizeAdjustPolicy">
                     <enum>QComboBox::AdjustToContents</enum>
                    </property>
@@ -1086,17 +1086,17 @@
                <item row="3" column="1">
                 <layout class="QGridLayout" name="gridLayout_10">
                  <item row="0" column="2">
                   <widget class="QLabel" name="editor_label_ac_threshold">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
-                    <string>Number of characters before list is shwon: </string>
+                    <string>Number of characters before list is shown: </string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QSpinBox" name="editor_spinbox_ac_threshold">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
@@ -1298,17 +1298,17 @@
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
-            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color and background color (for the latter, the color pink (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color, and background color (for the latter, the color magenta (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
@@ -1818,17 +1818,17 @@
                <property name="title">
                 <string>Colors for variable attributes </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="cb_hide_tool_tips">
                <property name="text">
-                <string>Hide tool tip</string>
+                <string>Hide tool tips</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
@@ -2000,17 +2000,17 @@
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
-                <string>Edit an actual shortcut by double clicking into the related cell</string>
+                <string>Edit a shortcut by double-clicking in a cell</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="topMargin">
                 <number>0</number>
                </property>
